packages/admin/src/cli.ts

```diff
@@ -281,6 +315,126 @@ yargs
       console.log(`Endpoint ID: ${endpointId}`);
     }
   )
+  .command(
+    'set-whitelist-expiration',
+    'Sets whitelist expiration of a user for the Airnode–endpoint pair',
+    {
+      ...airnodeRequesterRrpAuthorizerCommands,
+      ...mnemonicCommands,
+      airnodeAddress,
+      expirationTimestamp,
+    },
+    async (args) => {
+      const airnodeRequesterRrpAuthorizer = await evm.getAirnodeRequesterRrpAuthorizerWithSigner(
+        args.mnemonic,
+        args.derivationPath,
+        args.providerUrl,
+        args.airnodeRequesterRrpAuthorizer
+      );
+      await admin.setWhitelistExpiration(
+        airnodeRequesterRrpAuthorizer,
+        args.airnodeAddress,
+        args.endpointId,
+        args.userAddress,
+        args.expirationTimestamp
+      );
+      console.log(`Whitelist expiration: ${args.expirationTimestamp}`);
```

Comment:
It is the unix timestamp just because it is the same value the user of the cli provided as arg but I don't have a strong against transforming it to a more readable date string or maybe show both values 👍🏻